src/api/flaskr/service/profile/profile_manage.py

```
@@ -168,23 +197,39 @@ def get_profile_item_definition_list(
+                    Variable.shifu_bid.in_([parent_id, ""]),
+                    Variable.deleted == 0,
+                )
+                .order_by(Variable.id.asc())
+                .all()
+            )
+            if definitions:
+                return [
+                    convert_variable_definition_to_profile_item_definition(item)
+                    for item in definitions
+                ]
+        except Exception as exc:  # pragma: no cover - defensive fallback
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)